frontend/src/global_styles/layout/_toolbar.sass

```diff
@@ -167,6 +167,9 @@ $nm-color-success-background: #d8fdd1
     @include toolbar-element-sizing
     border: 1px solid var(--toolbar-item--border-color)
 
+    &.-no-padding
+      padding-right: 0
```

Comment:
I see multiple issues with such a rule:

* A class named `-no-padding` should remove all padding not only the right side.
* Such generic class are usually helpful for other places as well and should not be limited to the toolbar only.

Reply:
I checked other place, seems it's due to using different `icon_dropdown` and `spot-icon spot-icon_dropdown` lead such problem, I rename to more BEM style like, I have to admin it's not perfect, but at least better than No fix the spacing problem.

Feel free to propose a better solution for me, I'm more than glad to close this PR.